pretix_sepadebit/payment.py

```diff
@@ -114,6 +118,33 @@ def settings_form_fields(self):
                                  'configuring at least 7 days.'),
                      min_value=1
                  )),
+                ('earliest_due_date',
+                 forms.DateField(
+                     label=_('Earliest debit due date'),
+                     help_text=_('Earliest date the direct debit can be due. '
+                                 'This date is used as the direct debit due date, if the order date plus pre-notification time would result in a due date earlier than this.'
+                                 'Customers with orders using the earliest due date, will receive an email informing them about the upcoming due date pre-notification days beforehand.'),
+                     required=False,
+                     widget=forms.widgets.DateInput(attrs={'class': 'datepickerfield'})
+                 )),
+                ('mail_payment_reminder_subject',
+                 I18nFormField(
+                     label=_("Due date reminder subject"),
+                     help_text=_('The subject of the notification email. '
+                                 'You can use the tags {creditor_id}, {creditor_name}, {account}, {iban}, {bic}, {reference} and {due_date}. '
```

Comment:
Hmm, is there a reason I can't use the standard and plugin-provided placeholders such as ``name`` or ``event``? We usually auto-generate this help text, you can look for inspiration on how to do this in a plugin e.g. here:
https://github.com/pretix-unofficial/pretix-vacc-utils/blob/main/pretix_vacc_autosched/forms.py#L117

Also, I think the amount would be one of the major important things in such an email :)